base_layer/core/src/validation/header/header_full_validator.rs

```diff
@@ -41,6 +41,14 @@ use crate::{
 
 pub const LOG_TARGET: &str = "c::val::header_full_validator";
 
+// 999785e3bb6a43189c7236de7c9720df27d56f6b9724cd22f5115a097d3f770e is a bad block that bypassed validation due to a bug
+// in saving Monero seeds. The block uses a randomX  VM key of
+// 91ef83186cefaa646dc4c6e950e68e4debab52b4f4a9b7f465891e91fe5f6ce4, this key was used between height 2729 and 843 which
+// is about what you would expect from the Monero consensus. This block reuses this key at height 26320, which is a
+// violation of the Monero consensus rules. But in order to keep the network on the same chain, we whitelist this block
+// to bypass validation as that is the only validation it has failed.
+pub const WHITELISTED_HEADERS: [&str; 1] = ["999785e3bb6a43189c7236de7c9720df27d56f6b9724cd22f5115a097d3f770e"];
```

Comment:
```suggestion
#[cfg(nextnet)]
pub const WHITELISTED_HEADERS: [&str; 1] = ["999785e3bb6a43189c7236de7c9720df27d56f6b9724cd22f5115a097d3f770e"];
```

Reply:
i.e. let's only add this exception for nextnet. I would also be satisfied with a comment 
```
[
// Nextnet height 22222
"99978..."
]
```